app/views/development_metrics/_products_list.erb

```diff
@@ -13,7 +13,7 @@
         <div data-turbolinks-permanent class="collapse" id="<%= product.name %>-collapse">
             <ul class="btn-toggle-nav list-unstyled fw-normal pb-1">
               <% product.repositories.each do |repository|%>
-                <a href='<%= (repositories_development_metrics_path({repository_name: repository.name, metric: { from: initial_from, to: initial_to, }})) %>' class='btn btn-toggle sidebar_font' role='button'><%= repository.name %></a>
+                <a href='<%= (repositories_development_metrics_path({repository_name: repository.name, metric: { from: initial_from, to: initial_to, }})) %>' class='btn btn-toggle sidebar_font' role='button'><%= repository.friendly_name ? repository.friendly_name : repository.name%></a>
```

Comment:
WDYT about adding a method in the model like `(friendly_name.presence || name)` instead of this ternary?

Reply:
thanks!!